vault/token_store.go

```diff
@@ -335,12 +335,12 @@ func NewTokenStore(c *Core, config *logical.BackendConfig) (*TokenStore, error)
 
 				Fields: map[string]*framework.FieldSchema{
 					"urlaccessor": &framework.FieldSchema{
-						Type:        framework.TypeString,
-						Description: "Accessor of the token (URL parameter)",
+						Type:        framework.TypeCommaStringSlice,
+						Description: "Accessor(s) of the token (URL parameter)",
```

Comment:
The URL parameter can't be used in this way, and it's also deprecated. Just the request body is sufficient.